jmclient/test/test_wallet.py

```diff
@@ -285,6 +290,19 @@ def test_gettimelockaddress_method(setup_wallet, timenumber, locktime_string):
 
     assert addr == addr_from_method
 
+
+@freeze_time("2021-01")
+def test_gettimelockaddress_in_past(setup_wallet):
+    jm_single().config.set("BLOCKCHAIN", "network", "mainnet")
```

Comment:
Shouldn't we revert this change at the end of the function?

Reply:
the config is (re)loaded for every test case in the `setup_wallet` fixture

Reply:
Oh, yes. My bad.